language-design.md

```diff
@@ -27,8 +41,17 @@ How would you know a well-designed language? What are the symptoms? How would
 you know a poorly designed language? What are the symptoms?
 
 **Response**
+I think one of the symptoms of a well designed language is that, as Guy Steele says, "new words defined by a library should look just like primitives of the language." In other words, the language should be seamless.
 
+I think a symptom of a poorly designed language is that there are many exceptions to the language rules. -Daniel
 
+I would agree that the potential to grow seamlessly is quite important, because it can facilitate faster growth in languages. After all, "Lisp grew much faster than APL did, because many users could try things out and put their best code out there for other users to use and to add to the language." [Steele 1998] And the faster a language grows, potentially anyway, the sooner the icky bits get dealt with. One might determine whether a language is "seamless" by looking, but this isn't
+completely foolproof and depends on what is classified as a primitive. For example, if everything in the Java standard library is a primitive, then Java is a pretty seamless language, but if not, then the standard library and all user libraries look a bit different than the primitives.
+
+I would also say that intuitiveness is important as well (though I have to say that Quorum, as mentioned in the the Pavlus article, is intuitive because it's verbose, and I'm sure it would be a pain to actually write code in). Based on the large response to the proposed change in CSS in the Verou post, I would imagine that many people other than myself value intuitive languages; a language that is not intuitive will probably not get used very much and as a consequence won't grow much either.
+
+One symptom of a poorly designed language is if it is distinctly not intuitive (although I admit this is by no means an objective measure). Even if a language can (or even does) grow, I would not consider it a well-designed language if it does things that don't make any sense (looking at [you](https://bugs.php.net/bug.php?id=45647) [PHP](https://bugs.php.net/bug.php?id=41523)) (for those who don't feel like following the link, the date/time 0000-00-00 00:00:00 is an error, but the
+date/time 00-00-00 00:00:00 is actually November 30, 1999. I would not expect this, and would not consider it a good design decision.) -Aaron
```

Comment:
I agree with the sentiment that both seamlessness and intuitiveness of a language are crucial. In particular, I liked the PHP example Aaron provided. One quality Rebekah and I mentioned found in a well-designed language that Steele seemed to stress a lot was user involvement in the development of the language. User involvement in the development of language seems to translate to scalability. Because you both seemed to approach the question from the user perspective (how a user may benefit from a well-designed language), it would have been nice to see some mention of how these benefits translate to the user's ability to contribute and to allow for the language to grow.